Lib/test/test_ctypes/test_keeprefs.py

```diff
@@ -98,6 +100,29 @@ def test_p_cint(self):
         x = pointer(i)
         self.assertEqual(x._objects, {'1': i})
 
+    @unittest.expectedFailure  # gh-46376, gh-107131, gh-107940, gh-108222
+    def test_pp_ownership(self):
```

Comment:
Is it possible to instead have a huge `expectedFailure`, to correctly detect *where* it is failing instead? Because among the `assert*` calls, I don't know which of these is the one that should actually fail (same question for all the other expected failures).